### ISSUE TITLE
Right brace sign not spoken at the correct punctuation level while using Tamil language

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5152]
**Reported by td.dinakar on 2015-06-13 05:56**
In the punctuation level "none" and "some", the right brace } sign is read even though the sign is set to speak only in the level "most" in the symbols.dic file. 

This happens only when Tamil language is selected in the voice settings dialog. There is no problem when English language is used. 

In both the Tamil and English symbols.dic file, the right brace sign is set to speak only at the level "most". 

I am using NVDA 2015.2 rc2 release. This problem was there even in earlier versions.

**Comment by ehollig** [comment]
Closing, because This seems to work correctly for me using NVDA Master-14590,5efe71c8.

1. Open Notepad and type, "This is a test }"
2. Ensure you are using ESpeak with the Tamil voice and Punctuation/symbol level is set to most in the Voice settings dialog.
3. Read the line in the notepad file. Notice the } is read.
4. Change the Punctuation/symbol level in voice settings to some and read the line again.
Notice the } is not read.
Note: I also checked the Punctuation/symbol pronunciation and the } was set to most.
